Python/PDF_Reader/README.md

```diff
@@ -0,0 +1,25 @@
+# PDF_Reader
+
+At times, we feel like having an assistant which could read our lengthy documents for us and we could keep listening to it. This script is very useful in such cases. Whenever the user specifies the path to the file in their system, and execute this program, the pdf is read aloud.
+
+## Setup instructions and Guidelines
+
+1. Install Python 3.x (recommended) from <a href="https://www.python.org/downloads/">here</a>

```

Comment:
Installing python seems a little redundant, try removing this and we'll be good to go

Reply:
✔ Done I removed it